DEVELOPER.md

```diff
@@ -54,6 +54,13 @@ M3 has an extensive, and ever increasing, set of tests to ensure we are able to
 
 Once you have identified a change you want to make, and gathered consensus by talking to some devs, go ahead and make a branch with the changes. To test your changes:
 
+(0) If necessary, generate GoMock files
+
+```shell
+# Replace xyz with the package you want to generate files for, e.g. dbnode
+make mock-gen-xyz
```

Comment:
Maybe also add that you can run for all top level packages by omitting the suffix?
```
# Generate mocks for all top level packages
make mock-gen
```

Reply:
Done